src/plugins/packages/index.js

```diff
@@ -39,34 +39,40 @@ function loadPackages() {
 		if (err) {
 			return;
 		}
+
 		packages.forEach((packageName) => {
-			const packageFile = getModuleInfo(packageName);
-			if (!packageFile) {
-				return;
-			}
-			packageMap.set(packageName, packageFile);
-			if (packageFile.type === "theme") {
-				themes.addTheme(packageName, packageFile);
-			}
-
-			if (packageFile.onServerStart) {
-				packageFile.onServerStart(packageApis(packageName));
-			}
+			fs.stat(Helper.getPackageModulePath(packageName), function(err2, stat) {
+				if (err2 || !stat.isDirectory()) {
+					return;
```

Comment:
Can we stick in a debug warning for package not existing, because it could be useful for some errors people might have at some point.

Reply:
Done.